buildSrc/subprojects/parser/src/test/java/org/apache/calcite/test/DialectGenerateTest.java

```diff
@@ -461,7 +461,7 @@ private void assertFileProcessed(String testName, boolean specifyFilePath) {
   @Test public void testUnparseNonReservedKeywordsMultiplePartitions() {
     ExtractedData extractedData = new ExtractedData();
     DialectGenerate dialectGenerate = new DialectGenerate();
-    for (int i = 0; i < 501; i++) {
+    for (int i = 0; i < 251; i++) {
```

Comment:
nit: Can we reference the static field in DialectGenerate here? e.g. through a public getter.